### ISSUE TITLE
Provide --rm-module to remove existing module or remove jsp/jstl as default module in docker image

### ISSUE THREAD

**Issue by jnehlmeier** [eclipse/jetty.project#10224]
**Jetty version(s)**
_[Jetty 9.x is now at End of Community Support](https://github.com/eclipse/jetty.project/issues/7958)_

Docker image 10.0.15 (probably just all images)

**Enhancement Description**

I am currently trying to migrate a custom built jetty docker image to the official one so I have to deal with less surprises like the recent introduction of quotes on parameters.

It seems like the official docker image adds JSP and JSTL support by default and I am wondering why you do that. Shouldn't downstream projects decide which features to enable? I don't use JSP/JSTL at all and would like to remove it. But it seems there is no `--rm-module` as the opposite of `--add-module`. So the only option I have is to figure out what the JSP/JSTL modules have added to jetty base and revert these modifications manually.

Any chance you would remove JSP/JSTL as default modules or provide a `--rm-module` parameter to do the cleanup work based on the description in `*.mod` file?

**Comment by joakime** [comment]
Just delete `${jetty.base}/start.d/jsp.ini` and/or `${jetty.base}/start.d/jstl.ini`

**Comment by jnehlmeier** [comment]
Yeah that is what I do now. Same for websocket which pulls in two implementations while new projects likely only need the javax version.

**Comment by sbordet** [comment]
This still seems to be relevant.

**Comment by lachlan-roberts** [comment]
The Jetty docker images for `12.0.x`+ now only include the following modules by default:
`server,http,ext,resources`, so no JSP or JSTL anymore

so this can be closed
